### PR TITLE
Remove dropwizard-servers, replace functionality by JerseyServers

### PR DESCRIPTION
**Pull request by uschi2000**

### CODE REVIEW

File: jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/JerseyServers.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+public final class JerseyServers {
+    private JerseyServers() {}
+
+    /**
+     * Server-side stacktraces are serialized and transferred to the client iff {@code serializeStacktrace} is {@code
+     * true}. Configures a Jersey server w.r.t. http-remoting conventions: registers tracer filters and
+     * exception mappers.
+     */
+    public static void configure(
+            final ResourceConfig jersey, ExceptionMappers.StacktracePropagation stacktracePropagation) {
```
Comment (markelliot) [inline jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/JerseyServers.java]: `jersey` parameter does not need to be final.

File: jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/ExceptionMappers.java [hunk 2]
```diff
@@ -16,6 +16,8 @@
 
 package com.palantir.remoting1.servers.jersey;
 
+import com.google.common.collect.ImmutableList;
+import java.util.List;
 import javax.ws.rs.ext.ExceptionMapper;
 
 public final class ExceptionMappers {
```
Comment (markelliot) [inline jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/ExceptionMappers.java]: worth just combining this class into the `JerseyServers` class? Can still keep the `getExceptionMappers` method, just consolidate classes?

File: jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/JerseyServers.java (hunk 1 above)
Comment (uschi2000) [reply jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/JerseyServers.java]: done

File: jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/ExceptionMappers.java (hunk 2 above)
Comment (uschi2000) [reply jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/ExceptionMappers.java]: done

**Review by markelliot — APPROVED** [review]

**Pull request merged**